src/Kount/Util/Khash.php

```diff
@@ -52,7 +52,7 @@ public static function getSaltPhrase() {
   public static function hashPaymentToken ($token) {
     $firstSix = mb_substr($token, 0, 6, 'latin1');
     $hash = self::hash($token, 14);
```

Comment:
are `mb_substr` and `hash` null-safe in terms of received string value? what is going to return `mb_substr(null, 0, 6, 'latin1');`? `null` or an error will be thrown?